Update libraryList.js

diomed opened the pull request:
upped

Foundation - 6.5.0-rc3
FontAwesome - 5.4.1.
Uikit 3  - 3.0.0-rc.16 | always the latest
Bulma - 0.7.2
d3 - v5

chinchang commented:
Hi @diomed Thanks again! I left some questions on your commit in this PR, mainly on the domain changes in this PR.
![screenshot 2018-10-15 15 40 24](https://user-images.githubusercontent.com/379918/46944811-b11aed00-d090-11e8-948e-d123252677e4.png)

diomed commented:
I switched from cloudflare to jsdeliver because that's the cdn Foundation uses in their documentation.
---
I also switched from cloudflare to getuikit.com source because they have mad updating habit,
every week another version. This won't stabilize until they get full release. 
This way it's always in the last rc version. 
[at the moment, that's 19]
